Library/ViewModels/ChangeEmailViewModel.swift

```diff
@@ -257,6 +241,5 @@ ChangeEmailViewModelOutputs {
 }
 
 private func shouldEnableSaveButton(email: String?, newEmail: String, password: String) -> Bool {
```

Comment:
Do we allow space-only passwords? Otherwise we might want to trim these too:

```suggestion
private func shouldEnableSaveButton(email: String?, newEmail: String, password: String) -> Bool {
  guard email != newEmail else { return false }

  return ![newEmail, password]
    .map { !$0.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty }
    .contains(false)
}
```